static/app/components/profiling/ProfilingOnboarding/profilingOnboardingModal.tsx

```diff
@@ -404,8 +404,8 @@ function AndroidInstallSteps({
             {`<application>
   <meta-data android:name="io.sentry.dsn" android:value="${dsn}" />
   <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
-  <meta-data android:name="io.sentry.traces.profiling.enable" android:value="true" />
-</application>`}
+  <meta-data android:name="io.sentry.traces.profiling.sample-rate" android:value="1.0" />
+  </application>`}
```

Comment:
@JonasBa i intentionally deleted the spaces here so it aligns with the opening `<application>` tag above

Reply:
Ah, let me fix it.